host/src/app.c

```diff
@@ -390,6 +390,10 @@ static bool captureStop(void)
 bool captureGetPointerBuffer(void ** data, uint32_t * size)
 {
   PLGMPMemory mem = app.pointerShapeMemory[app.pointerShapeIndex];
+  if(!mem)
+  {
```

Comment:
formatting, no need for braces for a single if statement.
```c
if (x)
  something();
```

Reply:
Mixed up work standards and LG standards. Should be resolved!